lib/cylc/task_remote_mgr.py

```diff
@@ -41,7 +41,6 @@
 
 
 REC_COMMAND = re.compile(r'(`|\$\()\s*(.*)\s*(`|\))$')
-REMOTE_INIT_FAILED = 'REMOTE INIT FAILED'
 
 
 class TaskRemoteMgmtError(Exception):
```

Comment:
I expect that this should subclass something from `cylc.flow.exceptions`. Probably `SuiteConfigError`.

Reply:
Dam, I missed that one.

Yes should probably subclass something, not that important, not related to this work.